Assets/Scripts/Utility/RMBLayout.cs

```diff
@@ -349,39 +349,40 @@ public static GameObject CreateBaseGameObject(ref DFBlock blockData, int layoutX
                     -obj.YPos + blockFlatsOffsetY,
                     obj.ZPos + BlocksFile.RMBDimension) * MeshReader.GlobalScale;
 
-                // Import custom 3d gameobject instead of flat
-                if (MeshReplacement.ImportCustomFlatGameobject(obj.TextureArchive, obj.TextureRecord, billboardPosition, flatsParent) != null)
-                    continue;
+                bool isImported;
+                GameObject go = MeshReplacement.ImportCustomFlatGameobject(obj.TextureArchive, obj.TextureRecord, billboardPosition, flatsParent);
+                if (!(isImported = go))
```

Comment:
Is this meant to be an assignment or a comparison? I'd prefer this is split out for clarity.

Reply:
Done.